spec/features/collection_drafts/forms/hidden_fields_spec.rb

```diff
@@ -0,0 +1,16 @@
+# MMT-1571
+describe 'Removed field', js: true do
```

Comment:
Does this test work without `js: true`?
Where possible we should run our tests without js. Loading it adds extra time to our tests.